pkg/provider/azure/azclients/azclients.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package azclients
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/storage/armstorage"
+	storageservice "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/service"
+	"sigs.k8s.io/cloud-provider-azure/pkg/azclient"
+	"sigs.k8s.io/cloud-provider-azure/pkg/azclient/configloader"
+)
+
+type Impl struct {
+	azureConfig *AzureConfig
+
+	storageAccountsClient    *armstorage.AccountsClient
+	blobContainersClient     *armstorage.BlobContainersClient
+	managementPoliciesClient *armstorage.ManagementPoliciesClient
+	blobServiceClient        *storageservice.Client
+}
+
+func (azclients *Impl) getTokenCredential() (azcore.TokenCredential, error) {
```

Comment:
"This" in Go should generally be the first character of the type in lowercase. Here, it should be `i`.

Reply:
but `Impl` probably isn't the right name for type

Reply:
My first name of this struct is AzclientsImpl, but go lint says it's already under package azclients, and azclients.AzclientsImpl looks duplicated, so I update the name to Impl only. Let me find a better name instead.

Reply:
Updated to Azclients underlay pkg/provider/azure/clients module